infrastructure/scaffold/networking.tf

```diff
@@ -1,22 +1,7 @@
 locals {
```

Comment:
why not use a data source? If someday we recreate the vpc/subnets, we'll need to change this. Also, data source would help us identify those subnets easier